### PR TITLE
Iss 211 proj9

### LINKED ISSUE #211: improve projection docs
imrpove https://merginmaps.com/docs/gis/proj/

- add that we now support TIF shift (After https://github.com/MerginMaps/input/pull/2199 is released)
- maybe add a link to https://github.com/OSGeo/PROJ-data where the TIF grids can be found
- if https://github.com/MerginMaps/qgis-mergin-plugin/issues/444 is fixed, it can refer to this updated page for how to solve validation error
- also mention HOW to add custom data-grid file to QGIS desktop app -> downlaod and copy the shift to proj/share folder

### LINKED ISSUE #218: Datum shift grids support
Datum shift grids are packaged when creating project and validated before sync.
https://github.com/MerginMaps/qgis-mergin-plugin/pull/455

### PR DESCRIPTION
**Pull request by alex-cit**
fix #211 
fix #218 
https://merginmaps.com/docs/gis/proj/ elaborated to include more info and instructions on how to work with grid shift files in Input
- added links to further reading (related to #100)

### CODE REVIEW

File: src/gis/proj.md [hunk 1]
```diff
@@ -1,13 +1,55 @@
...
+   Grid shift files installed in QGIS on your computer can be found in these locations:
+   - Windows users: `C:\Users\USER\AppData\Roaming\QGIS\QGIS3\profiles\default\proj` (USER is your username)
+   - macOS users: `~/Library/Application\Support/QGIS/QGIS3/profiles/default/proj`
+   - Linux users: `~/.local/share/QGIS/QGIS3/profiles/default/proj`
+   :::
+   
+3. Synchronise the project using <QGISPluginName />
+4. Download the project with the grid shift files to <MobileAppName />. 
+   When the project is downloaded to the app for the first time, you will get a **PROJ Error** that will prompt you to **restart the app** to load the grid shift files. The error message should not appear after the restart of the app.
+![input shift grid](./input-custom-prj.png)
```
Comment (wonder-sk) [inline src/gis/proj.md:55]: In the upcoming release of Mergin plugin, this should be simplified - the plugin should check for any grids used by the project automatically and offer an automatic fix if the grid file is missing (will copy the grid(s) to `proj` sub-dir). Also if a new project is created in the wizard, grids will get packaged automatically. So maybe the points 1+2 can be kept as a manual procedure for older plugin version users (or a fallback for those who want to do things manually)

see https://github.com/MerginMaps/qgis-mergin-plugin/pull/455

File: src/gis/proj.md [hunk 2]
```diff
@@ -1,13 +1,55 @@
...
-
 <Badge text="Since Input 0.8.0" type="info"/>
+[[toc]]
+
+## Custom projections in QGIS
+It is common to have multiple layers with different coordinate reference systems in one QGIS project. To display these layers accurately, it is necessary to define the transformation between the coordinate reference systems. 
+
+QGIS contains the basic set of projections and transformations, which are handled by [PROJ](https://proj.org). However, some specific projections may require you to install additional grid shift files to get accurate transformations of coordinates. If this is the case, QGIS will show you a warning and will allow you to install the grid shift file from a folder.
+
+![](./proj_transformation_installation.png)
```
[the remark this answers is not in the document]
Comment (alex-cit) [reply src/gis/proj.md:10]: Oops, I accidently opened the project in QGIS 3.16. Thanks for noticing!

**Comment by alex-cit** [comment]
fix #211 fix #218 
- added grid support in plugin (todo: check the plugin version when plugin is released)
- fixed screenshot

**Comment by PeterPetrik** [comment]
@varmar05 this is already released no (proj stuff in input), so we can merge this one?

**Comment by varmar05** [comment]
> @varmar05 this is already released no (proj stuff in input), so we can merge this one?

yes, should be already released

**Pull request merged**